internal/blocksync/pool.go

```diff
@@ -177,12 +179,26 @@ func (pool *BlockPool) removeTimedoutPeers() {
 	pool.sortPeers()
 }
 
+func weBlockTheChain(state sm.State, myAddr crypto.Address) bool {
```

Comment:
I think this could stay at the reactor side, not at the pool.

Reply:
I'm thinking of making it part of `ValidatorSet`, as we may need it in node as well (to get rid of `onlyValidatorIsUs` later)

Reply:
This is good.